documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -6,11 +6,18 @@
 = Kafka consumer configuration tuning
 
 [role="_abstract"]
-Use a basic consumer configuration with optional properties that are tailored to specific use cases.
-
+Use configuration properties to optimize the performance of Kafka consumers.
+You can use standard Kafka consumer configuration options.
```

Comment:
TBH, this sentence seems weird. What are the `standard options`? Is this expected to refer to the options that Kafka supports? Or some subset of important or common options? I would probably leave this out.